custom-fonts.md

```diff
@@ -23,6 +41,71 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+### Second use the font
+
+Second use the font by delcaring the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html)
+constant and adding the font family used in the `pubsec.yaml`.
+
+```
+// declare the text style
+const textStyle = const TextStyle(
+  fontFamily: 'Raleway',
+);
+
+// use the text style
+var buttonText = const Text(
+  "Use the font for this text",
+  style: textStyle,
+);
+```
+
+## Using the Included Material Design fonts
+If you're looking at using the Material Design fonts,
```

Comment:
"looking at using" is a bit too colloquial :)

Do you have a more matter-of-fact way of saying this?